webui/react/src/routes/utils.ts

```diff
@@ -130,10 +131,62 @@ export const linkPath = (aPath: string, external = false): string => {
 };
 
 export const paths = {
+  cluster: (): string => {
+    return '/cluster';
+  },
+  dashboard: (): string => {
+    return '/dashboard';
+  },
+  docHyperparameterSearch: (): string => {
+    return '/docs/reference/experiment-config.html#searcher';
+  },
+  docIndex: (): string => {
+    return '/docs';
+  },
+  docReleaseNotes: (): string => {
+    return '/docs/release-notes.html';
+  },
+  docRestApi: (): string => {
+    return '/docs/rest-api/';
+  },
+  docUsers: (): string => {
+    return '/docs/topic-guides/users.html';
+  },
```

Comment:
**non-blocking:** don't have a good answer, but if the docs list grows any longer, it might just be best to have a general `docs` path where it will just take a suffix from the caller. For example:
```
docs: (suffix?: string): string => {
  return `/docs${suffix}`;
},
```